apps/console/src/extensions/i18n/resources/en-US/extensions.ts

```diff
@@ -3037,7 +3037,20 @@ export const extensions: Extensions = {
                     error: {
                         description: "Error updating the attribute as an unique attribute. Please try again.",
                         message: "Error updating claim"
+                    },
+                    success: {
+                        description: "Successfully updated the {{claimName}} uniqueness validation scope to across user stores.",
```

Comment:
`to across user stores.` Shall we improve this content?

Reply:
Suggestion: `Successfully updated the {{claimName}} uniqueness validation scope.`

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/7106/commits/32fec6a5960c62780009b2d358bae81f319c021c